src/AspNet.Security.OAuth.HubSpot/HubSpotAuthenticationOptions.cs

```diff
@@ -0,0 +1,32 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+using static AspNet.Security.OAuth.HubSpot.HubSpotAuthenticationConstants;
+
+namespace AspNet.Security.OAuth.HubSpot;
+
+/// <summary>
+/// Defines a set of options used by <see cref="HubSpotAuthenticationHandler"/>.
+/// </summary>
+public class HubSpotAuthenticationOptions : OAuthOptions
+{
+    public HubSpotAuthenticationOptions()
+    {
+        ClaimsIssuer = HubSpotAuthenticationDefaults.Issuer;
+        AuthorizationEndpoint = HubSpotAuthenticationDefaults.AuthorizationEndpoint;
+        TokenEndpoint = HubSpotAuthenticationDefaults.TokenEndpoint;
+        UserInformationEndpoint = HubSpotAuthenticationDefaults.UserInformationEndpointFormat;
+        CallbackPath = HubSpotAuthenticationDefaults.CallbackPath;
+
+        ClaimActions.MapJsonKey(ClaimTypes.Name, "user");
+        ClaimActions.MapJsonKey(ClaimTypes.Email, "user");
```

Comment:
This doesn't seem right to me - email addresses aren't people's names and vice-versa (maybe apart from some celebrities' babies). Did you mean to use `ClaimTypes.NameIdentifier`?